docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of how Temporal works. 
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks.
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions.
+It does so by automatically reconstructing its previous state.
```

Comment:
To a technical person, "reconstructing its previous state" might be confusing or unclear why that helps make an application more reliable. 

Maybe it would be helpful to use the analogy of comparing Temporal’s state recreation to a saved game in a video game that lets you pick up where you left off after a crash. Or a paused movie.
Example: "Temporal keeps track of everything your application does. So if something goes wrong, like a power outage, Temporal helps the application pick up right where it left off, just like resuming a video game right after a crash." 

I would also  emphasize the complexity and challenges involved in state management. For example, "Manual state management often requires careful handling of concurrency, ensuring data consistency, and managing failures during the save or reload process, all of which add significant overhead to the development process" because someone non-technical might wonder why saving the state is a complex thing to do

Reply:
Good points.